config/dev1/sys.config

```diff
@@ -7,7 +7,6 @@
       {internal, [
           {swagger_port, 3113},
           {websocket, [ {port, 3114},
-                        {handlers, 100},
```

Comment:
Removing this leads to either using the YAML setup or [using the hardcoded default value](https://github.com/aeternity/epoch/blob/master/apps/aehttp/src/aehttp_app.erl#L14). I like it better this way.